easy_profiler_core/include/easy/serialized_block.h

```diff
@@ -91,7 +91,6 @@ namespace profiler {
 
     public:
 
-        CSwitchEvent() = default;
```

Comment:
removes
```bash
serialized_block.h:94:9: warning: explicitly defaulted default constructor is implicitly deleted [-Wdefaulted-function-deleted]
        CSwitchEvent() = default;
        ^
serialized_block.h:88:39: note: default constructor of 'CSwitchEvent' is implicitly deleted because base class 'profiler::Event' has a deleted default constructor
    class PROFILER_API CSwitchEvent : public Event
                                      ^
/details/profiler_public_types.h:117:9: note: 'Event' has been explicitly marked deleted here
        Event() = delete;
```